lb/fetchCharacters.js

```diff
@@ -0,0 +1,7 @@
+export async function fetchCharacter(url) {
+  const response = await fetch(url);
```

Comment:
I understand the parameter, but it makes this function a bit more flexible than it is named. I'd suggest making the function less versatile